.github/workflows/rules.yml

```
@@ -9,37 +9,9 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
         python-version: [3.7, 3.8, 3.9, '3.10']
-    runs-on: ${{ matrix.os }}
-    steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
-      with:
-        python-version: ${{ matrix.python-version }}
-    # Use ``abstrevol`` branch of qibo temporarily
-    - name: Install package
-      run: |
-        python -m pip install --upgrade pip
-        pip install pylint
-        pip install pytest-cov
-        pip install git+https://github.com/qiboteam/qibo
-        pip install .
-    - name: Test with pylint
-      run: |
-        pylint src -E -d E1123,E1120
-    - name: Test with jit enabled
-      run: |
-        pytest --pyargs qibojit
-    - name: Test with jit disabled
-      if: startsWith(matrix.os, 'ubuntu')
-      run: |
-        NUMBA_DISABLE_JIT=1 pytest --cov=qibojit --cov-report=xml --pyargs qibojit
-    - name: Upload coverage to Codecov
-      if: startsWith(matrix.os, 'ubuntu')
-      uses: codecov/codecov-action@v2
-      with:
-        token: ${{ secrets.CODECOV_TOKEN }}
-        file: ./coverage.xml
-        flags: unittests
-        name: codecov-umbrella
-        fail_ci_if_error: true
+    uses: qiboteam/workflows/.github/workflows/rules.yml@adjqibojit
```

Since the PR in workflows is approved, I'd wait for that one to merged before merging this
```suggestion
    uses: qiboteam/workflows/.github/workflows/rules.yml@main
```